### DISCUSSION TITLE
Need help with anything and everything

### DISCUSSION THREAD

**Discussion by Rossy0666** [community/community#124292] [New to GitHub]
Can anyone help me figure out what exactly I'm doing

**Comment by serdar-webdev** [comment]
Can you describe your situation?

**Reply by Rossy0666** [reply]
Yeah I have no clue what I'm doing on here

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @Rossy0666 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
